### PR TITLE
buildsys: put "automatically generated" marker as comment

### PR DESCRIPTION
**Pull request by masatake**
Signed-off-by: Masatake YAMATO <yamato@redhat.com>

### CODE REVIEW

File: autogen.sh [hunk 1]
```diff
@@ -1,7 +1,10 @@
 #!/bin/sh
 
 ctags_files=`make -f makefiles/list-translator-input.mak`
-misc/dist-test-cases > makefiles/test-cases.mak && \
+{
+	echo "# Generated by $0"
+	misc/dist-test-cases
```
Comment (b4n) [inline autogen.sh]: shouldn't the comment rather be emitted by `misc/dist-test-cases` itself?

**Comment by masatake** [comment]
@b4n, You are right.

**Comment by b4n** [comment]
Off topic: *misc/dist-test-cases* uses `&&` at every line *end*, but no `\`, that's odd.  Shouldn't this script simply `set -e` and drop the `&&`s?

**Comment by masatake** [comment]
Yes, `-e` can be used.

> but no `\`,
I don't understand it well. What do you mean?
As far as I know `&&` at the end of line is enough for chaining condition statements.
Am I wrong ?

**Comment by b4n** [comment]
> As far as I know `&&` at the end of line is enough for chaining condition statements.
Am I wrong ?

Oh, no you're right, but I didn't know that :)  So I learned something new today :)
So my comment is moot then, the script is fine.

**Comment by masatake** [comment]
I see.

**Pull request merged**